### PR TITLE
[NeoDays/RetroDays] 17 item images by Delcan

### PR DESCRIPTION
**Pull request by Delcan314**
#### Summary
NeoDays "17 item images by Delcan"

#### Content of the change

Contains new sprites for the following items: Adobe mortar
Baselard
Bone shiv
Set of butterfly swords
Clamp
Fire brick
Horse tack
Shillelagh
Loaded stick
Kettle
Mattress
Grip hook
Deck of playing cards
Pipe cleaner
Pipe fittings
Sandbox kit
Steel sewing awl

This is my first pull request to add to CDDA, and is hopefully the first of many.
#### Testing

image
image

### CODE REVIEW

File: gfx/NeoDays/pngs_tiles_10x10/items/adobe_mortar_0.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"id": "mortar_adobe", "fg": ["3089_adobe_mortar_0"], "bg": []}
```
Comment (int-ua) [inline gfx/NeoDays/pngs_tiles_10x10/items/adobe_mortar_0.json:1]:
```suggestion
{"id": "mortar_adobe", "fg": "3089_adobe_mortar_0"}
```

were these files produced by some script? You can turn arrays into the string when they have only one element and drop empty `bg` values completely

**Comment by int-ua** [comment]
Is there a reason to keep numbers in filenames? @Fris0uman

**Comment by Fris0uman** [comment]
Nop, no real reason, just to remember that the files are from the decompose script and also that it would be a pain to rename them all.

File: gfx/NeoDays/pngs_tiles_10x10/items/adobe_mortar_0.json (hunk 1 above)
Comment (Delcan314) [reply gfx/NeoDays/pngs_tiles_10x10/items/adobe_mortar_0.json:1]: Files were produced by carefully edited copypasta - I can leave out empty "bg" information in the future.  My apologies, my most significant programming experience was ZZT-OOP like 25 years ago and I'm always reluctant to make changes to things that might end up being necessary.

**Pull request closed without merging**